applications/crossbar/src/modules/cb_rates.erl

```diff
@@ -22,6 +22,7 @@
 
 -define(PVT_FUNS, [fun add_pvt_type/2]).
 -define(PVT_TYPE, <<"rate">>).
+-define(NUMBER_RATE, <<"number_rate">>).
```

Comment:
How do you feel about `/rates/number/+74956063602`. We typically try not to put verbs in the URI path